content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -70,6 +70,113 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 
 {{< /note >}}
 
+### Enable dual-stack via kubeadm
+
+Install kubeadm following the steps from the [Installing Kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) documentation.
+
+Make sure that nodes allow IPv6 forwarding, if not, run `sudo sysctl -w net.ipv6.conf.all.forwarding=1` on every node in the cluster.
+
+{{< note >}}
+`kubeadm upgrade` will change `IPv6DualStack` to true by default if feature gate is not set in old cluster. However, cluster-cidr and service-cidr modification are not supported.
+{{< /note >}}
+
+#### Create a dual-stack cluster
```

Comment:
Perhaps this could live inside https://kubernetes.io/docs/setup/

Reply:
Moved